### DISCUSSION TITLE
Piramit küp sayısı programı

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus.github.io#66] [Soru Cevap]
![image](https://user-images.githubusercontent.com/52765796/158380519-f49232c3-eafe-4500-afe5-575130abc492.png)

Küplerden oluşan bir piramitin birinci katında 1, ikinci katında 3, üçüncü katında 6 küp bulunmaktadır. Şayet bu piramit 100 katlı olsaydı toplam kaç küp bulunacaktır?

**Comment by TahaGuzell** [comment]
```
def toplam_kup_sayisi(kat_sayisi):
    toplam_kup = 0
    for kat in range(1, kat_sayisi + 1):
        toplam_kup += kat * (kat + 1) // 2 
    return toplam_kup

kat_sayisi = 100

sonuc = toplam_kup_sayisi(kat_sayisi)
print(kat_sayisi,"katlı piramitte toplam",sonuc,"küp bulunur.")


```
